.changeset/clean-guests-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ai-sdk/core': major
```

Comment:
[q] I think this should be `major` as this will be part of v2 release notes. Am I right?

Reply:
Technically speaking, this should follow semver based on this specific change. So here it should not be major. But when we do bump, changeset takes the highest from the changeset files.